src/main/java/core/basesyntax/dao/FruitDao.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax.dao;
+
+import java.util.Objects;
+
+public class FruitDao {
```

Comment:
The `FruitDao` class seems unnecessary as per the previous review. Consider removing it to avoid confusion and adhere to the task requirements.